src/components/newsletter/newsletter-sign-up.tsx

```diff
@@ -47,6 +47,8 @@ const newsletterSignUpStyles = css`
   padding: 4.5rem 0.75rem;
   background-color: var(--taupe);
 
+  font-size: 18px;
```

Comment:
can you make this `1.125rem`? I've been careful everywhere to keep it `1.125rem`